_posts/2016-07-19-expanding-our-collection-of-transit-data-in-italy.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+category: news
+published: false
+isThereTitle: true
+title: Expanding our collection of transit data in Italy
+---
+
+!["a train in Torre Pellice in the Piedmont region of Italy, photo by Antonio Martinetti"](https://c2.staticflickr.com/2/1510/24314365009_252b45739e_z.jpg)
```

Comment:
If the license allows, please copy the image file into this repo so we can reference a local copy.